packages/salesforcedx-vscode-apex/src/index.ts

```diff
@@ -316,7 +315,10 @@ async function createLanguageClient(extensionContext: vscode.ExtensionContext) {
     }
 
     languageClientUtils.setClientInstance(languageClient);
-    const handle = languageClient!.start();
+
+    void resolveAnyFoundOrphanLanguageServers();
```

Comment:
The void is b/c the function is async but we don't want to wait on it correct?

Reply:
Yes, fire and forget!

Reply:
If there is a more "vscode" way to do this, please let me know.

Reply:
nope I think that's the way 👍 It's just fairly unusal.  Typically we wait for all the things.